test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -57,16 +62,24 @@ def send_message(self, message: str):
         
     # Verifications region:        
     def verify_last_message_sent(self, message: str):
-        [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
-        verify(loaded, "Checking last message sent: " + message)
-        verify_text_contains(str(last_message_obj.text), str(message))
+        # Get the message text
+        # We don't search for StatusTextMessage_chatText directly, because there're 2 instances of it in a reply message
+        last_message_obj = self.get_message_at_index(0)
+        text_message_obj = getChildrenWithObjectName(last_message_obj, "StatusMessage_textMessage")[0]
+        text_edit_obj = getChildrenWithObjectName(text_message_obj, "StatusTextMessage_chatText")[0]
+        verify(not is_null(text_edit_obj), "Checking last message sent: " + message)
+        verify_text_contains(str(text_edit_obj.text), str(message))
 
     def verify_last_message_sent_is_not(self, message: str):
-        [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
-        if not loaded:
+        chat_log_obj = wait_and_get_obj(ChatComponents.CHAT_LOG.value)
+        last_message_obj = chat_log_obj.itemAtIndex(int(0))
+        text_message_objs = getChildrenWithObjectName(last_message_obj, "StatusMessage_textMessage")
+        if len(text_message_objs) == 0:
             test.passes("Success: No message was found")
```

Comment:
Not related to this pr but seen now.. It should be a call to the `SquishVerificationDriver` not a direct call to squish api. Could you kindly update it?